apps/web/google_docs.talon

```diff
@@ -0,0 +1,326 @@
+browser.host: docs.google.com
+-
+
+tag(): user.line_commands
+
+file menu | menu file: key(ctrl-alt-f)
+open: key(cmd-o)
+[open] (revision | version) history [open]: key(cmd-alt-shift-h)
+print: key(cmd-p)
+save: key(cmd-s)
+edit menu | menu edit: key(ctrl-alt-e)
+paste without formatting: key(cmd-shift-v)
```

Comment:
I believe this is called `paste match` already:

https://github.com/knausj85/knausj_talon/blob/4d39608dc9979017d7a0218b195feb54ff136ffa/core/edit/standard.talon#L14

You may need to override the `edit.paste_match_style` action, like

https://github.com/knausj85/knausj_talon/blob/4d39608dc9979017d7a0218b195feb54ff136ffa/core/edit/edit_mac.py#L142-L143